### DISCUSSION TITLE
Custom domain issues.

### DISCUSSION THREAD

**Discussion by CampbellHarry** [community/community#68517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello i have recently set my domain back to the origin https://campbellharry.github.io/ it was previously http://hdev.me/ however if you click on the first link it takes me to hdev.me though the custom domain has been removed and so has the cname any help with this please?

**Comment by CampbellHarry** [comment]
Just to add to this when I'm on a incognito tab it works

**Comment by vishalkrishna8** [comment, marked as answer]
Hello @CampbellHarry ,
If your old custom domain is still redirecting to your GitHub Pages site after removing it, Here's what you can do to resolve this:

1. Clear your browser cache and cookies.
2. Flush your computer's DNS cache (instructions vary by OS).
3. Wait for DNS changes to propagate (may take hours).
4. Check your domain registrar for lingering DNS records.
5. Ensure your GitHub repository settings have no custom domain.
6. If the issue persists, contact GitHub Support for help.

Let me know if this was helpful!

**Reply by CampbellHarry** [reply]
Hiya @vishalkrishna8 Found the problem it was due to the cashe just reset it and it worked a treat thanks
